maud_extras/meta.rs

```diff
@@ -0,0 +1,80 @@
+extern crate maud;
```

Comment:
I think this should be merged into `lib.rs` -- there isn't enough here to justify a separate module.